src/main/java/org/scijava/parallel/Node.java

```diff
@@ -0,0 +1,41 @@
+package org.scijava.parallel;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public abstract class Node<T> {
```

Comment:
Can this file be moved to a dedicated branch? It is not used at the moment at all. It was intended to be used for defining dependencies between individual tasks, but this feature has not been implemented yet.

Also, this is probably my fault, sorry for this.

Reply:
I suggest not to complicate it and leave it on this branch.

Reply:
OK